Bitkit/Models/BackupPayloads.swift

```diff
@@ -12,7 +12,7 @@ struct WalletBackupV1: Codable {
 struct MetadataBackupV1: Codable {
     let version: Int
     let createdAt: UInt64
-    let tagMetadata: [ActivityTagsMetadata]
+    let tagMetadata: [PreActivityMetadata]
```

Comment:
should we name it `preActivityMetadata`? I can do the same in Android ;)

Or even `activityMetadata`, since we won't likely have a different activity metadata soon 🤷🏻